modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -46,9 +46,11 @@ baremetal   4.10.12   True        False         False      3d15h
 +
 [source,terminal]
 ----
-$ oc delete bmh -n openshift-machine-api vmaster-0
-$ oc delete machine -n openshift-machine-api kni1-master-0
+$ oc delete bmh -n openshift-machine-api <host_name>
+$ oc delete machine -n openshift-machine-api <machine_name>
 ----
++ 
+Replace `<host name>` with the name of the host and `machine name` with the name of the machine. The machine name appears under the `CONSUMER` field
 +
 After you remove the `BareMetalHost` and `Machine` objects, then the `Machine` controller automatically deletes the `Node` object.
```

Comment:
Is `machine` an object is this context? like Node object or should it read as  "  then the machine controller...."

Reply:
Thanks, fixed this!

Reply:
Thank again, I have removed the back ticks as it's not an object.